.gitpod.yml

```
@@ -0,0 +1,6 @@
+ports:
+- port: 6052
+  onOpen: open-preview
+tasks:
+- before: python setup.py install
+  command: python esphome my_configuration.yaml dashboard
```

```suggestion
  command: python esphome config dashboard
```

The first parameter for the dashboard command is where the configs should be stored, not a file (config/ is also ignored in .gitignore)